backend/src/main/java/ai/verta/modeldb/entities/ExperimentEntity.java

```diff
@@ -108,6 +115,12 @@ public ExperimentEntity(Experiment experiment) throws InvalidProtocolBufferExcep
   @OrderBy("id")
   private CodeVersionEntity code_version_snapshot;
 
+  @Column(name = "workspace")
+  private String workspace;
+
+  @Column(name = "workspace_type")
+  private Integer workspace_type;
+
```

Comment:
If we should have to implement the experiment own workspace logic then these new fields we have to implement otherwise we should revert this.

Reply:
reverted